jupyter_collaboration/handlers.py

```diff
@@ -56,6 +59,7 @@ class DocumentRoom(YRoom):
     def __init__(self, type: str, ystore: BaseYStore, log: Optional[Logger]):
         super().__init__(ready=False, ystore=ystore, log=log)
         self.type = type
+        self.session_id = str(uuid.uuid4())
```

Comment:
What is this for? It doesn't seem to be used.

Reply:
Ouch, good catch. That's a left over form some experiments I did.